### PR TITLE
CLIs name inconsistencies

### PR DESCRIPTION
**Pull request by AntoineRondelet**
- Add some missing flags to CLIs
- Stick to Kebab case for all CLIs for consistency (including for executable bash scripts)
- Add missing binaries to the PATH
- Add missing shebang
- Extend the CODING_STANDARDS to document the naming conventions for executables and structure in conditional tests.

### CODE REVIEW

File: libzeth/core/bits.hpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ template<size_t numBits> class bits : public std::array<bool, numBits>
 
     /// Construct from initializer-list.
     // cppcheck-suppress noExplicitConstructor
-    template<typename... boolList> bits(const boolList &...bits);
+    template<typename... boolList> bits(const boolList &... bits);
```
Comment (AntoineRondelet) [inline libzeth/core/bits.hpp:38]: Hum... odd, running `./scripts/format` locally seems to add this space which seems to break the format running on the CI. Need to have a look into this.

Comment (dtebbs) [reply libzeth/core/bits.hpp]: Worth checking the version of clang-format. (I'm using version 11).

File: CODING_STANDARDS.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ New code should follow these rules, and old code should be changed over time to
 - Each file should have a well-defined purpose and be concerned with a single narrow concept.
 - Add unit tests for new code. As far as possible, one test per source file or logical group of files.
 - Names should be as clear and concise as possible, avoiding opaque abbreviations.
+- In conditions, the tested variable must appear on the left of the operator. This is to stay consistent with English, which writes/reads from left to right.
+  ```
+  // GOOD:
+  if (age >= 18) { // Reads "If age is above 18" (the test is on the value of `age` here, so it appears first)
```
Comment (dtebbs) [inline CODING_STANDARDS.md:14]: Could be worth moving / shortening / reformatting the comment to make it a bit more readable in a code box.

**Review by dtebbs — COMMENTED** [review]
In order to make this uniform, we'll need to update all references in scripts etc, and we should also consider covering scripts themselves (the command name should not depend on the language used to author it).

However, this conflicts with https://github.com/clearmatics/neodizk/pull/36#discussion_r691299416.

**Comment by AntoineRondelet** [comment]
> In order to make this uniform, we'll need to update all references in scripts etc, and we should also consider covering scripts themselves (the command name should not depend on the language used to author it).
> 
> However, this conflicts with [clearmatics/neodizk#36 (comment)](https://github.com/clearmatics/neodizk/pull/36#discussion_r691299416).

Good point I didn't take the scripts into consideration here, was focusing on our C++ and Python CLIs. So we have even more inconsistencies then.. All our file names are `snake_case` (bash, python, c++) so all bash scripts have a `_` as separator. However, all commands from the CLIs (except `prover_server`) are in `kebab-case`, so it looks like the way to go is to use `snake_case` everywhere then.

File: CODING_STANDARDS.md (hunk 2 above)
Comment (AntoineRondelet) [reply CODING_STANDARDS.md]: Restructured to a multiple line comment

**Comment by AntoineRondelet** [comment]
I get an infinite loop on the CI because `server_start` keeps waiting for the `prover-server` to be active. Have a look closer to the CI scripts, I must have forgotten some renamings here.

**Comment by dtebbs** [comment]
> I get an infinite loop on the CI because `server_start` keeps waiting for the `prover-server` to be active. Have a look closer to the CI scripts, I must have forgotten some renamings here.

Looks like some missing renamings in the ci_utils.sh file  (e.g. line 111).

**Review by dtebbs — APPROVED** [review]
LGTM

**Comment by AntoineRondelet** [comment]
> > I get an infinite loop on the CI because `server_start` keeps waiting for the `prover-server` to be active. Have a look closer to the CI scripts, I must have forgotten some renamings here.
> 
> Looks like some missing renamings in the ci_utils.sh file (e.g. line 111).
> 
> UPDATE: fixed in the latest commit

Thanks @dtebbs

**Pull request merged**